src/main/java/org/opensrp/service/formSubmission/EventsListener.java

```diff
@@ -140,7 +142,7 @@ private synchronized void addServerVersion() {
 				for (Event event : events) {
 					try {
 						Thread.sleep(1);
-						event = eventService.processOutOfArea(event);
+						event = eventService.processOutOfArea(event,username);
```

Comment:
You can pass `event.getProvider` as username here

Reply:
Passing event.getProviderId() as username